.travis.yml

```diff
@@ -90,10 +90,10 @@ jobs:
             - libomp
           update: false
 
-    - name: macOS 10.14, llvm 10
+    - name: macOS 10.14, latest llvm (version 11)
```

Comment:
Can we drop `(version 11)`? Otherwise this would be misleading once llvm 12 is released.

Reply:
I think the one way or the other, we have to update the `.travis.yml` regularly to keep track of the compiler changes.